README.md

```diff
@@ -32,6 +32,7 @@ git clone https://github.com/devitocodes/devito.git
 cd devito
 conda env create -f environment.yml
 source activate devito
+pip install -r requirements.txt
```

Comment:
That is not necessary, the environment.yml contains all the requirements and installs it

Reply:
Thanks @mloubout we're having a discussion here: https://github.com/devitocodes/devito/issues/1065